docs/api-rollup.js

```diff
@@ -0,0 +1,42 @@
+/*!
```

Comment:
Nit: it might be worth renaming this to include the "local" nature of it. I would probably also note in the docs that this sort of simulates the `download` process done for API docs from releases, but for local code. I would also note that users need to build the repo before running this.